app/src/main/res/layout/fragment_new_search.xml

```diff
@@ -144,9 +144,11 @@
         android:text="@string/search_not_found"
         android:textAppearance="@style/UiKit.TextAppearance.Regular.Text3"
         android:textColor="@color/text_night"
+        android:textSize="17sp"
```

Comment:
textSize should be from UiKit and TextAppearance

Reply:
ohh man you are right! damn!